platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/fingerprint/ConfigurationCacheFingerprintWriter.kt

```diff
@@ -181,6 +183,7 @@ class ConfigurationCacheFingerprintWriter(
     var closestChangingValue: ConfigurationCacheFingerprint.ChangingDependencyResolutionValue? = null
 
     init {
+        buildScopedContext.runWriteOperation { write(host.buildInvocationScopeId) }
```

Comment:
💭 Shouldn't this information be part of `EntryDetails` rather than "the 1st fingerprint entry"?

Reply:
Done, PTAL!